x-pack/platform/plugins/shared/onechat/public/application/hooks/use_conversation_scroll_actions.ts

```diff
@@ -112,7 +112,9 @@ export const useConversationScrollActions = ({
   // Scrolls the most recent round to the top of it's parent scroll container
   const scrollToMostRecentRoundTop = useCallback(() => {
     if (!scrollContainer) return;
-    scrollToMostRecentRound({ scrollContainer, position: 'start' });
+    requestAnimationFrame(() => {
+      scrollToMostRecentRound({ scrollContainer, position: 'start' });
+    });
```

Comment:
Since the conversation input refactor we've needed this I think - and I'm not sure why. Something to do with the ordering of rendering/painting to the DOM. We should **not** be calling `scrollToMostRecentRound` until the next time we paint to the DOM I.e. after the next render. We need the render to complete so that the `conversation_round` has the minimum height of 1 full screen height so that there is space for us to scroll it to the top.